### ISSUE TITLE
[Bug] Huge file size on Windows

### ISSUE THREAD

**Issue by heftymouse** [WerWolv/ImHex#1939]
### Operating System

Windows

### What's the issue you encountered?

ImHex is massive on Windows compared to other platforms. It unpacks to around 460 megabytes.

### How can the issue be reproduced?

1. Download ImHex for Windows, either portable or installer
2. Observe unexpectedly large file size

### ImHex Version

1.35.4

### ImHex Build Type

- [ ] Nightly or built from sources

### Installation type

MSI

### Additional context?

I believe this is in large part because debug information is bundled with the program. Many binaries have DWARF sections included and others have PDBs in the root directory. An extreme example of this is libpl.dll where DWARF sections seem to take up around 144 MB.

**Comment by iTrooz** [comment]
See https://github.com/WerWolv/ImHex/issues/1714

TLDR; We know, nothing planned to fix it for now (mostly because of lack of time), if you have/can work on a solution that would still allows us to get crash stacktraces, please tell us
